cdci_data_analysis/pytest_fixtures.py

```diff
@@ -352,27 +367,78 @@ def follow_output():
 
     service=url_store[0]
 
-    yield service
+    return dict(
+        url=service, 
+        pid=p.pid
+        )        
 
-    
-    print("will keep service alive a bit, for async")
-    time.sleep(0.5)
+@pytest.fixture
+def dispatcher_long_living_fixture(pytestconfig, dispatcher_test_conf_fn):
```

Comment:
I like the idea! But where do you use this fixture? In place of `dispatcher_live_fixture`?

Reply:
yes, behaves exactly the same. Just we need to be cautious that this way dispatcher initialization is not tested every time, and there could be some accumulating issues. So it's best to reserve it for really demanding tests.